js/options.js

```diff
@@ -77,8 +83,12 @@ function onRestore() {
     $('reloadUnpinnedOnly').checked = settings.reloadUnpinnedOnly == true
     $('reloadAllLeft').checked = settings.reloadAllLeft == true
     $('reloadAllRight').checked = settings.reloadAllRight == true
+    $('closeAllLeft').checked = settings.closeAllLeft == true
+    $('closeAllRight').checked = settings.closeAllRight == true
     $('reloadStartup').value = (typeof settings.reloadStartup == 'undefined') ? 'none' : settings.reloadStartup
-    $('bypassCache').checked = settings.bypassCache == true
+    $('bypassCache').checked = settings.bypassCache == true,
+	$('buttonDefaultAction').value = (typeof settings.buttonDefaultAction == 'undefined') ? 'window' : settings.buttonDefaultAction

```

Comment:
```suggestion
    $('buttonDefaultAction').value = (typeof settings.buttonDefaultAction == 'undefined') ? 'window' : settings.buttonDefaultAction
```
nit: space instead of tab

Reply:
updated in my fork